backend/homework/8/src/main/java/com/example/springhandson3/controller/VehicleController.java

```diff
@@ -0,0 +1,56 @@
+package com.example.springhandson3.controller;
+
+import com.example.springhandson3.dto.RequestDTO;
+import com.example.springhandson3.dto.ResponseDTO;
+import com.example.springhandson3.service.InventoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class VehicleController {
```

Comment:
Add a mapping here /vehicle

And it'll help you to avoid adding the same 'vehicle' at all endpoints

then your endpoints will be
.../
.../{id}
.
.
.../getLeastExpensive